### ISSUE TITLE
make loadSystemDBSpan aware of intents

### ISSUE THREAD

**Issue by petermattis** [cockroachdb/cockroach#2871]
`loadSystemDBSpan` currently doesn't handle write intents. When it encounters one it prints an error like:

```
E1021 09:56:18.583287 53015 storage/replica.go:1296  could not load SystemDB span: conflicting intents on ["\xff\v\x03\v\x01\v\x04\v\x02"]: resolved? false
```

This doesn't seem right. This situation lasts until someone else comes in and resolves the intent.

**Comment by veteranlu** [comment]
@petermattis Has this issue been solved? 
the write intents either be committed or uncommitted, do u want to resolve the committed intent(`handleSkippedIntents` do the job), or get the older version from storage which intent uncommitted, why not scan with `CONSISTENT` mode?

**Comment by petermattis** [comment]
I thought @tschottdorf fixed this.

**Comment by tschottdorf** [comment]
yeah, fixed in 670ffc61.
